commitlintplugins.test.ts

```diff
@@ -348,11 +348,28 @@ test('proper-issue-refs3', () => {
     let commitMsgWithHashtagRefInBlock =
         "foo: this is only a title" + "\n\n" + "Bar baz:\n\n```\ntype Foo = string #123\n```";
     let properIssueRefs3 = runCommitLintOnMsg(commitMsgWithHashtagRefInBlock);
-    console.log('HERE =======>' + properIssueRefs3.stdout)
     expect(properIssueRefs3.status).toBe(0);
 });
 
 
+test('proper-revert-message1', () => {
+    let commitMsgWithoutProperRevertMessage = 
+        'Revert "add abbreviations.ts"\n\n' + 
+        'This reverts commit 0272f587c7eece147e8d1756116b0b43e11c34ac.';
+    let properRevertMessage1 = runCommitLintOnMsg(commitMsgWithoutProperRevertMessage);
+    expect(properRevertMessage1.status).not.toBe(0);
+});
+
+
+test('proper-revert-message2', () => {
+    let commitMsgWithProperRevertMessage = 
+        'Revert 9bb557e54830690b8a8e403d1b74780d86b07b4c\n\n' + 
+        'We need to revert this commit, because/otherwise bla bla.'
```

Comment:
let's make the commit message identical to the one in the first test, except for adding the reasoning later; we don't need the user to re-write the revert commit message, we want the user to **expand** on it

Reply:
> I'd like to see the commit "set defaultIgnores to false" be green after a red one (so that it's clear that it's what makes the test pass), but I can't see that:
> 
> <img alt="Screenshot 2022-11-15 at 12 58 33 PM" width="474" src="https://user-images.githubusercontent.com/331303/201830272-6d56346e-6d1a-4baa-ac89-f6d85acf5f0a.png">

Oh, you are right. It's a squashed branch, and the previous branch was far behind the master and had conflicts with the master branch so I had to rebase it. Therefore it can't be seen now. What should I do?